api/app/Modules/Library/Events/Saves/SavedTrackRemoved.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Modules\Library\Events\Saves;
+
+use App\Modules\Core\Events\HasUserContext;
+use App\Modules\Library\Events\UserAction;
+
+class SavedTrackRemoved extends UserAction
+{
+    use HasUserContext;
+
+    public string $id;
```

Comment:
We don't need an `id` here. We already have the `userId` from the `UserAction`, and the `trackId` below.